friends/src/components/Login/loginStyles.css

```diff
@@ -0,0 +1,19 @@
+.loginForm {
+  display: flex;
+  flex-direction: column;
+  width: 258px;
+  margin: auto;
+}
+
```

Comment:
I appreciate the bit of effort you put into actually adding some style to your login, not everyone did this.